### DISCUSSION TITLE
allow multiple legal names

### DISCUSSION THREAD

**Discussion by VladimirAlexiev** [eprocurementontology/eprocurementontology#548] [Ideas]
Currently Organization admits only `[0..1]` legal names.
However, there are countries with multiple official languages (eg BE) or scripts (eg SR), so it should allow multiple.

If you can, add a restriction `sh:uniqueLang` (see https://www.w3.org/TR/shacl/#UniqueLangConstraintComponent), which means "only one name per lang". It's the same restriction as on `skos:prefLabel`

**Comment by AchillesDougalis** [comment]
In ePO 4.0.0  the org:Organization concept allows  for multiple legal names. 
![image](https://github.com/OP-TED/ePO/assets/143995053/571dd07a-0c45-45bd-ad97-b2fb6786efb2)
This issue will be moved to discussions so we can look further into the second part about `sh:uniqueLang`.
